src/Nox.Generator/Presentation/Api/OData/EntityControllerGenerator.cs

```diff
@@ -200,7 +197,7 @@ private static void GeneratePrivateChildrenGetById(NoxSolution solution, EntityR
     private static void GenerateChildrenGetById(NoxSolution solution, EntityRelationship relationship, Entity child, Entity parent, CodeBuilder code)
     {
         code.AppendLine($"[EnableQuery]");
-        code.AppendLine($"[HttpGet(\"api/{parent.PluralName}/{PrimaryKeysAttribute(parent)}/{relationship.Name}/{PrimaryKeysAttribute(child, "relatedKey")}\")]");
+        code.AppendLine($"[HttpGet(\"{solution.Infrastructure.Endpoints.ApiRoutePrefix}/{parent.PluralName}/{PrimaryKeysAttribute(parent)}/{relationship.Name}/{PrimaryKeysAttribute(child, "relatedKey")}\")]");
```

Comment:
Line 684 there is one more place that have to use the ApiRoutePrefix

Reply:
fixed